isis/src/qisis/apps/ipce/IpceMainWindow.cpp

```diff
@@ -211,31 +212,47 @@ namespace Isis {
 
     if ( qobject_cast<SensorInfoWidget *>(newWidget) ||
          qobject_cast<TargetInfoWidget *>(newWidget) ||
+         qobject_cast<ControlHealthMonitorView *>(newWidget) ||
          qobject_cast<TemplateEditorWidget *>(newWidget)) {
+      dock->setAllowedAreas(Qt::LeftDockWidgetArea | Qt::RightDockWidgetArea);
       splitDockWidget(m_projectDock, dock, Qt::Vertical);
+
+      // Save view docks for cleanup during a project close
+      m_specialDocks.append(dock);
     }
     else {
+      // The following logic not guaranteed to work as intented. If user moves one of the "special"
+      // views such as sensor from below the project dock to the right side of project, the following
+      // will put the new dock to the right of the moved "special" dock instead of tabbing.  The only
+      // way to possibly insure the intended functionality would be to check for the position of the
```

Comment:
I guess I'm not clear (from the text) as to what the "intended" functionality is